lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -255,6 +257,13 @@ func (handler *ContainersHandlersImpl) RemoveContainerHandler(params containers.
 		case exec.RemovePowerError:
 			return containers.NewContainerRemoveConflict().WithPayload(&models.Error{Message: err.Error()})
 		default:
+			f, ok := err.(types.HasFault)
+			if ok {
```

Comment:
You can combine these two lines into one.  It's more go-like.

```
if f, ok := err.(types.HasFault); ok {
```

Reply:
yeah that's just an oversight on my part; will do